examples/GMW_N26W082_2020_v3/GMW_N26W082_2020_v3.json

```diff
@@ -39,7 +39,6 @@
     26.0
   ],
   "properties": {
-    "proj:code": "EPSG:4326",
```

Comment:
You _could_ keep it, as v1.2.0 allows both for forwards-compatability: https://github.com/stac-extensions/projection/releases/tag/v1.2.0. Not necessary, tho